### PR TITLE
bcda 7125: improve cclf/utils test coverage

### PR DESCRIPTION
**Pull request by karinamzalez**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-7125

## 🛠 Changes

- renames cclf/testutils to utils <-- testutils file name was ignored by sonarqube
- replaces calls to depracated ioutils with os equivalents
- adds TestInvalidFilePath

## ℹ️ Context for reviewers

- increases test coverage for cclf/utils by 1.57% (codecov)

## ✅ Acceptance Validation

- assert that all tests are passing

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: bcda/cclf/utils/cclfUtils_test.go [hunk 1]
```diff
@@ -46,6 +46,21 @@ func (s *CCLFUtilTestSuite) TestImportInvalidEnvironment() {
 	assert.EqualError(err, "invalid argument for environment")
 }
 
+func (s *CCLFUtilTestSuite) TestInvalidFilePath() {
+	assert := assert.New(s.T())
+	err := ImportCCLFPackage("improved-small", "test-partially-adjudicated", models.FileTypeRunout)
+	assert.EqualError(err, "unable to locate ../../../../../../shared_files/cclf/files/synthetic/test-partially-adjudicated/small in file path")
+}
+
+// func (s *CCLFUtilTestSuite) TestAddFileToZip() {
```
Comment (laurenkrugen-navapbc) [inline bcda/cclf/utils/cclfUtils_test.go:55]: should this be commented out here?

**Comment by laurenkrugen-navapbc** [comment]
I'm not sure how much more improvements we can make on coverage without refactoring, but I would like to get the rest of the teams feedback to see if they agree/disagree! Can you and Alex and Kevin as reviewers as well? ⭐

**Comment by alex-dzeda** [comment]
> I'm not sure how much more improvements we can make on coverage without refactoring, but I would like to get the rest of the teams feedback to see if they agree/disagree! Can you and Alex and Kevin as reviewers as well? ⭐

I think the only thing that jumps out at me as quick ways to increase coverage here would be to feed invalid file names (for example, including a newline/carriage return in the file name) but that would be marginal at best.

**Review by alex-dzeda — APPROVED** [review]
Looks good, created backlogged item BCDA-7468 to investigate non-deterministic test outputs in general. 🚀

**Review by kyeah — APPROVED** [review]
Looks good, thank you!

**Pull request merged**